Mathlib/Algebra/Category/ModuleCat/Algebra.lean

```diff
@@ -65,9 +65,7 @@ instance linearOverField : Linear k (ModuleCat.{v} A) where
   homModule M N := LinearMap.module
   smul_comp := by
     -- Porting note: this was automatic by `aesop_cat`
-    aesop_cat_nonterminal
-    rw [LinearMap.smul_apply, LinearMap.smul_apply, LinearMap.map_smul_of_tower]
-    rfl
+    intros; ext; erw [LinearMap.map_smul_of_tower]; rfl
```

Comment:
Why was `erw` not necessary after `aesop_cat_nonterminal` but is needed now? I would prefer a `dsimp; rw` over `erw` any day: `erw` is a sign something is wrong.

Reply:
I managed to avoid the `erw` with a combination of `dsimp` and `rw`.